### PR TITLE
Update Project.php and Membership.php

### PR DESCRIPTION
**Pull request by tvaughan73**
Add limit/offset so getIdByName method will work properly.

### CODE REVIEW

File: lib/Redmine/Api/Membership.php [hunk 1]
```diff
@@ -101,6 +101,30 @@ public function remove($id)
...
 
+    /**
+     * Delete membership of project by user id
+     *
+     * @param int $project_id id of project
+     * @param int $user_id id of user
+     *
+     * @return false|\SimpleXMLElement|string
+     */
+    public function removeMember($project_id, $user_id)
```
Comment (kbsali) [inline lib/Redmine/Api/Membership.php]: please follow the coding style : var in camelCase :)

File: lib/Redmine/Api/Membership.php [hunk 2]
```diff
@@ -101,6 +101,30 @@ public function remove($id)
...
+    {
+        $memberships = $this->all($project_id);
+
+        if (isset($memberships['memberships']))
```
Comment (kbsali) [inline lib/Redmine/Api/Membership.php]: try running php-cs-fixer on this file please, it should fix the cs errors
thanks!

Comment (rps-repo) [reply lib/Redmine/Api/Membership.php]: I have fixed the camel case vars.  I do not know how to use php-cs-fixer and don't have time to figure it out, sorry.

Comment (rps-repo) [reply lib/Redmine/Api/Membership.php]: OK, changes have been made.

**Pull request merged**